packages/travelmakers-design-core/src/components/ControlIndicator/stories/ControlIndicator.stories.tsx

```diff
@@ -0,0 +1,71 @@
+import React from "react";
+import { ControlIndicator } from "../ControlIndicator";
+
+export default {
+  title: "@travelmakers-design/core/Component/ControlIndicator",
+  component: ControlIndicator,
+  argTypes: {
+    color: {
+      defaultValue: "navy",
+      description: "ControlIndicator 컴포넌트의 색상을 정합니다.",
+      options: ["navy", "white"],
+      control: { type: "inline-radio" },
+    },
+    size: {
+      defaultValue: "small",
+      description: "ControlIndicator 컴포넌트의 색상을 정합니다.",
+      options: ["small", "large"],
+      control: { type: "inline-radio" },
+    },
+    totalPage: {
+      defaultValue: 1,
+      description: "ControlIndicator 컴포넌트의 토탈 page 수를 정합니다.",
+      table: {
+        type: {
+          summary: "number",
+        },
+      },
+      control: { type: "number" },
+    },
+    // [추가]: 컴포넌트 내에서 current page 를 관리해야 버튼 동작을 컨트롤 할 수 있어서 외부에서 current page를 입력 받는 것은 추후 조정 예정
```

Comment:
추후 할 예정의 경우 `TODO: ~~`같은 형태의 주석으로 통일하면 좋을 것 같습니다~!
```suggestion
    // TODO: [추가] 컴포넌트 내에서 current page 를 관리해야 버튼 동작을 컨트롤 할 수 있어서 외부에서 current page를 입력 받는 것은 추후 조정 예정
```

Reply:
넵 확인했습니다!